src/coreclr/vm/ilmarshalers.cpp

```diff
@@ -4371,10 +4371,10 @@ FCIMPL3(void, MngdNativeArrayMarshaler::ConvertContentsToNative, MngdNativeArray
     if (*pArrayRef != NULL)
     {
         const OleVariant::Marshaler* pMarshaler = OleVariant::GetMarshalerForVarType(pThis->m_vt, TRUE);
-        SIZE_T cElements = (*pArrayRef)->GetNumComponents();
+        SIZE_T cElements = 0;
```

Comment:
This seems like an unintentional behavioral change.

Reply:
Ah, the safemul doesn't necessarily happen for the variable. That's why initialization warns.

Reply:
What are the de facto advantages of Sweden and Finland joining NATO?